prime-router/pom.xml

```diff
@@ -503,6 +504,11 @@
             <artifactId>jackson-databind</artifactId>
             <version>${jackson.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <version>${jackson.version}</version>
+        </dependency>
         <dependency>
```

Comment:
I was getting weird intellisense errors until I added annotations to the pom file as well.